hydra_base/lib/project.py

```diff
@@ -125,19 +131,34 @@ def update_project(project, **kwargs):
 
     proj_i = _get_project(project.id, user_id, check_write=True)
 
-    proj_i.name = project.name
-    proj_i.description = project.description
+    if project.name:
+        proj_i.name = project.name
+
+    if project.description:
```

Comment:
I think it  would be better as:

```python
if hasattr(project, 'description') and project.description is not None:
```

Reply:
This isn't so simple, as the incoming objects will be JSONObjects, which may not have a 'description' atribute, but the 'project.description' will resolve to none. So this check isn't necessary